schema/alert.xsd

```diff
@@ -0,0 +1,23 @@
+<xs:schema xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  attributeFormDefault="qualified"
+  elementFormDefault="qualified"
+  targetNamespace="https://hyperview.org/hyperview-alert"
```

Comment:
Is there any kind of standard for defining nested/sub-namespaces? Using forward slashes (`/`) instead of dashes seems like it might be more appropriate.

Reply:
There's no standard for sub-namespaces. Technically, namespaces don't even need to be valid URLs. That became the convention in order to minimize conflicts between different groups putting out schemas.

Reply:
Thoughts on using slashes vs the dashes?

Reply:
I have no preference either way, so we can use slashes in the future. It is a bit of a pain to change the existing namespaces though while maintaining backwards compatibility with older versions of the Hyperview client that are out there